spec/integration/schema_parser_spec.rb

```diff
@@ -10,7 +10,15 @@
             "attributes": {
               "name": "About naming stuff and cache invalidation"
             }
-          }
+          },
+          "included": [
```

Comment:
I think this should rather be a 3rd case for the only main resource schema

1. valid body only main resource
2. valid body main resource + unknown included
3. invalid body for main resource

I'm also not yet 100% sure if case 2 should success or fail but prefer it to fail